### PR TITLE
Sixth lol ppp

### LINKED ISSUE #276: Write sixth PPP

### PR DESCRIPTION
**Pull request by holzeis**
resolves #276

### CODE REVIEW

File: docs/ppp/iteration-6-22112022.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+We made big steps towards our 0.2.0 release, which will be a workable solution supporting CFD trading on lightning.
+
+- Send Payments via lightning invoices
+- Bug fixing and UX improvements
+- Added some basic validations
+- Added persistence on the phone
+- Replacing mock data with real data
+- Show payment history
+- Added continuous deployment to our maker pipeline
```
Comment (bonomat) [inline docs/ppp/iteration-6-22112022.md:23]:
```suggestion
- Added continuous deployment to our testnet pipeline
```

File: docs/ppp/iteration-6-22112022.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+### Progress ✅
+
+We made big steps towards our 0.2.0 release, which will be a workable solution supporting CFD trading on lightning.
+
+- Send Payments via lightning invoices
+- Bug fixing and UX improvements
+- Added some basic validations
+- Added persistence on the phone
+- Replacing mock data with real data
+- Show payment history
```
Comment (bonomat) [inline docs/ppp/iteration-6-22112022.md:22]:
```suggestion
- Show payment history for on-chain and off-chain transactions
```

**Review by bonomat — APPROVED** [review]
Lookg good, thanks for the writeup

File: docs/ppp/iteration-6-22112022.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+### Plans 📆
```
Comment (bonomat) [inline docs/ppp/iteration-6-22112022.md:1]:
```suggestion

### Public Demo

We are pleased to invite everyone to a public demo on 23.11.2022 at 09:00am UTC. 
The demo will take place in [https://meet.jit.si/10101-demo](https://meet.jit.si/10101-demo). 

### Plans 📆
```

**Review by da-kami — APPROVED** [review]

**Comment by bonomat** [comment]
bors r+

I'm going ahead and publish it

File: docs/ppp/iteration-6-22112022.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+
+### Public Demo 📽
+
+We are pleased to invite everyone to a public demo on 23.11.2022 at 09:00am UTC. 
+The demo will take place in [https://meet.jit.si/10101-demo](https://meet.jit.si/10101-demo). 
+The demo will be recorded and shared afterwards.
+
+### Plans 📆
+
+It's going to be a short iteration until Thursday and we are in the final steps polishing our solution. We plan the following tasks to hand-in a strong candidate to be selected for the final round.
```
Comment (luckysori) [inline docs/ppp/iteration-6-22112022.md:10]:
```suggestion
It's going to be a short iteration until Thursday and we are in the final steps polishing our solution. We have planned the following tasks to hand-in a strong candidate to be selected for the final round.
```

File: docs/ppp/iteration-6-22112022.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+- Publish a blog post about our innovation and value proposition
+- Prepare the pitch presentation and video promoting our solution
+- Record a demo video showing off CFD trading end to end
+- Release version 0.2.0 as apk downloadable from our [github repository](https://github.com/itchysats/10101/releases)
```
Comment (luckysori) [inline docs/ppp/iteration-6-22112022.md:15]:
```suggestion
- Release version 0.2.0 as an APK downloadable from our [GitHub repository](https://github.com/itchysats/10101/releases)
```

File: docs/ppp/iteration-6-22112022.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+- Last improvements on UI/UX and bug fixing
+- Create lightning invoices to receive money
```
Comment (luckysori) [inline docs/ppp/iteration-6-22112022.md:17]:
```suggestion
- Create Lightning invoices to receive money
```

File: docs/ppp/iteration-6-22112022.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+- Wiring up close (settle) CFD
```
Comment (luckysori) [inline docs/ppp/iteration-6-22112022.md:18]:
```suggestion
- Wiring up CFD settlement
```

File: docs/ppp/iteration-6-22112022.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+### Progress ✅
+
+We made big steps towards our 0.2.0 release, which will be a workable solution supporting CFD trading on lightning.
```
Comment (luckysori) [inline docs/ppp/iteration-6-22112022.md:22]:
```suggestion
We made big steps towards our 0.2.0 release, which will be a working solution bringing CFD trading to Lightning.
```

File: docs/ppp/iteration-6-22112022.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+- Send Payments via lightning invoices
```
Comment (luckysori) [inline docs/ppp/iteration-6-22112022.md:24]:
```suggestion
- Send payments via Lightning invoices
```

File: docs/ppp/iteration-6-22112022.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+- Bug fixing and UX improvements
+- Added some basic validations
```
Comment (luckysori) [inline docs/ppp/iteration-6-22112022.md:26]:
```suggestion
- Added some basic user input validation
```

**Review by luckysori — APPROVED** [review]
Nice! Thanks for writing this. Just style things.

**Comment by bonomat** [comment]
Published, thanks for all the feedback :) 

https://makers.bolt.fun/story/ppps-week-6-10101--426

**Pull request merged**